docs/geedocs/5.2.5/answer/3481558.html

```diff
@@ -89,8 +89,15 @@ <h3>Commands</h3>
   <pre>
 <code><b>--no_historical_imagery </b></code></pre>
   <p><i>Default</i>. Uses normal imagery for the project.</p>
-  <h2><a name="63542"></a>geconfigureassetroot</h2>
+  <h3>Options</h3>
+  <pre>
+  <code><b>--maxlevel </b><i>level</i></code></pre>
+  <div class="alert"> Deprecated in release GEE 5.2.5 and higher. Use <code>--maxleveloverride</code></a> instead.</div>
+  <p><i>Optional</i>. Sets the maximum level for the imagery. Uses internal Fusion scale. Does not match the levels in the Fusion UI.</p>
```

Comment:
```suggestion
  <p><i>Optional</i>. Sets the maximum level for the imagery. Uses internal Fusion scale. Deprecated because it not match the levels specified in the Fusion UI.</p>
```

Reply:
So those other CLI utilities are all compiled from this one .cpp file.

Reply:
Also worth noting: that aside from the geaddtoimageryproject, those other CLI utilities do not have any documentation. So if you want those documented, they will need entire documentation written. Also the help text is only for geaddtoimageryproject (though it is (mostly) applicable to the others.